tests/acceptance/standard/test_hooks_processor.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from tests.acceptance import run_gitlabform, create_group, create_project
+
+
+class TestHooksProcessor:
+    def test_project_hooks_create_delete(self, gl, project, project_hook):
+        target = project.path_with_namespace
+        test_yaml = f"""
+            projects_and_groups:
+              {target}:
+                hooks:
+                  {project_hook.url}:
+                    push_events: {project_hook.push_events}
+                    merge_requests_events: {project_hook.merge_requests_events}
+            """
+
+        run_gitlabform(test_yaml, target)
+
+        created_hook = project.hooks.get(project_hook.id)
+        assert project_hook == created_hook
```

Comment:
I think we can add one more assertion about hooks count being `1`.

Another option could be add another hook in the test config with different attributes. That way the delete hook can remove one and confirm that only the selected hook was deleted.